homeassistant/components/gogogate2/common.py

```diff
@@ -0,0 +1,133 @@
+"""Common code for GogoGate2 component."""
+from datetime import datetime, timedelta
+import logging
+from typing import Callable, NamedTuple, Optional
+
+from gogogate2_api import GogoGate2Api, InfoResponse
+from gogogate2_api.common import Door, get_configured_doors
+
+from homeassistant.components.gogogate2.const import (
```

Comment:
Did you forget to `git add const.py`?

Reply:
No. const.py is present. Besides, the build would have failed if it were missing. A lot of the gogogate2 component depends on the const file.

Reply:
Apologizes, ended up being a disk issue with my git repo.